lib/schema_tools/modules/hash.rb

```diff
@@ -192,6 +189,16 @@ def parse_object(obj, field, prop, opts)
         res
       end
 
+      private
+
+      def parse_object?(obj, field)
+        if obj.respond_to?( field )
+          rel_obj = obj.send( field )
```

Comment:
NAB: should we delete the spaces around the parameters to be consistent with the style of the json_schema_tools code base?